gbdk-lib/examples/gb/banks_autobank/Makefile

```diff
@@ -34,20 +26,11 @@ make.bat: Makefile
 # -Wl-yoA      : Have makebin automatically calculate required number of ROM banks (otherwise use -Wl-yo4 in this example)
 # -Wl-ya4      : Use 4 RAM banks
 # -Wl-yt19     : Use MBC5 cartridge type
-$(BINS):	$(OBJS)
-	$(LCC) $(CFLAGS) -autobank -Wb-ext=.rel -Wb-v -Wl-yt19 -Wl-yoA -Wl-ya4 -o $(BINS) $(OBJS)
-
-
-# It's also possible to do all the compiling, autobanking and linking in a single call to lcc
-# with none of the other makefile "targets" above. You can try this target out by using "make onepass".
-# Notice that the input files are the .c and .asm sources instead of the compiled object files.
-onepass:
```

Comment:
The one-line alternate, example target should be kept. The purpose of the makefile is to help teach users what the possibilities are and how to use them.

Reply:
Same as above - perhaps just provide specific examples for makefiles

Reply:
Though IDK since its all gonna continue being reworked at some point